src/config.ts

```diff
@@ -36,13 +35,46 @@ const DEFAULT_CONFIG: any = {
       type: "low",
       center: {lat: 45, lng: -82},
       strength: 6
+    }
+  ],
+  gulf: [
+    {
+      type: "high",
+      center: {lat: 37, lng: -107.1},
+      strength: 15.5
+    },
+    {
+      type: "high",
+      center: {lat: 33, lng: -67.5},
+      strength: 9.54
     },
     {
       type: "low",
-      center: {lat: 47, lng: -60},
-      strength: 7
+      center: {lat: 39.15, lng: -91},
+      strength: 15
     }
-  ],
+  ]
+} as const;
+
+export const selectPressureSystems = (startLocation: string) => {
+  const locationSystems = pressureSystems[startLocation as keyof typeof pressureSystems] || pressureSystems.atlantic;
```

Comment:
I think you could eliminate the cast and tighten up some of the other types by defining a `type StartLocation = "atlantic" | "gulf"` somewhere and then using it to define types here and when defining `pressureSystems`, etc.
```suggestion
export const selectPressureSystems = (startLocation: StartLocation) => {
  const locationSystems = pressureSystems[startLocation] || pressureSystems.atlantic;
```